src/plugins/intel_gpu/src/graph/impls/onednn/primitive_onednn_base.h

```diff
@@ -53,10 +54,23 @@ struct typed_primitive_onednn_impl : public typed_primitive_impl<PType> {
         _attrs(attrs),
         _pd(pd) {
             _enable_profiling = config.get_property(ov::enable_profiling);
+            _scratchpad_md = _pd.scratchpad_desc();
+
             GPU_DEBUG_GET_INSTANCE(debug_config);
             GPU_DEBUG_IF(!debug_config->dump_profiling_data.empty()) {
                 _enable_profiling = true;
             }
+
+            GPU_DEBUG_IF(debug_config->verbose >= 5) {
```

Comment:
I'd propose to have `4` being max verbosity level

Reply:
Changed.